app/core/application.coffee

```diff
@@ -61,6 +60,16 @@ Application = {
     )
     store.commit('me/updateUser', me.attributes)
     store.commit('updateFeatures', features)
+    if me.showChinaRemindToast()
+      setInterval ( -> noty {
+        text: '你已经练习了一个小时了，建议休息一会儿哦'
+        layout: 'topRight'
+        type:'warning'
+        killer: false
+        timeout: 5000
+        }), 3600000  # one hour
```

Comment:
Nitpick: Indentation looks a little off here.

Reply:
thank you, will do better next time